examples/demo11.py

```diff
@@ -247,9 +247,11 @@ def main(argv):
         # We will need the image position as well, so use the wcs to get that
         image_pos = wcs.toImage(world_pos)
 
-        # We also need this in the tangent plane, which we call "world coordinates" here,
+        # We also need this in the tangent plane, which we call "uv coordinates" here,
         # since the PowerSpectrum class is really defined on that plane, not in (ra,dec).
-        uv_pos = affine.toWorld(image_pos)
+        world_center = wcs.toWorld(image_center)
+        u, v = world_center.project(world_pos)
+        uv_pos = galsim.PositionD(u/galsim.arcsec, v/galsim.arcsec)
```

Comment:
This change isn't related to the rest of this PR, but a couple of the systems in GitHub CI started failing.  I think it must be a change on their end (either architecture or compiler, etc.) since this code doesn't use the Silicon class at all.  
In exact math, these two ways of calculating uv_pos are equivalent, and on all the machines I have access to, they differed only at the 1.e-16 level, which was fine.  But on some of the ubuntu GHA machines, they differed at 1.e-13, which was enough to eventually propagate into some pixels being different at the 1.e-8 level, so the yaml and py versions of the images (dtype=float32) were not identical, as we require.
So I changed the demo11.py code to calculate uv_pos the same way the config layer does.  I don't thin there's any loss in pedagogy from this, so this seems fine to me.